server/src/SymbolScanner.ts

```diff
@@ -20,6 +15,7 @@ import type {
 } from './DefinitionProvider'
 
 import { logger } from './lib/src/utils/OutputLogger'
+import { DIRECTIVE_STATEMENT_KEYWORDS } from './lib/src/types/directiveKeywords'
```

Comment:
Why did you name them directive_statement? I think `include_keywords` is simpler.

Reply:
oh, because I saw they put the word `directive` after all three of them: https://docs.yoctoproject.org/bitbake/bitbake-user-manual/bitbake-user-manual-metadata.html#inherit-directive